consensus/qkchash/qkchash_test.go

```diff
@@ -126,3 +126,29 @@ func sealBlock(t *testing.T, q *QKCHash, h *types.RootBlockHeader) {
 	h.Nonce = block.IHeader().GetNonce()
 	h.MixDigest = block.IHeader().GetMixDigest()
 }
+
+func TestQKCHashPow(t *testing.T) {
+	assert := assert.New(t)
+	diffCalculator := consensus.EthDifficultyCalculator{AdjustmentCutoff: 1, AdjustmentFactor: 1, MinimumDifficulty: big.NewInt(10)}
+	header := &types.RootBlockHeader{Number: 1, Difficulty: big.NewInt(10)}
+	q := New(true, &diffCalculator, false, []byte{}, 10)
+	rootBlock := types.NewRootBlockWithHeader(header)
+	resultsCh := make(chan types.IBlock)
+	err := q.Seal(nil, rootBlock, nil, 1, resultsCh, nil)
+	assert.NoError(err, "should have no problem sealing the block")
+	block := <-resultsCh
+	// Correct
+	header.Nonce = block.IHeader().GetNonce()
+	header.MixDigest = block.IHeader().GetMixDigest()
+	err = q.VerifySeal(nil, header, big.NewInt(10))
+	assert.NoError(err, "should have correct nonce / mix digest")
+
+	// Wrong
+	header.Nonce = block.IHeader().GetNonce() - 1
+	err = q.VerifySeal(nil, header, big.NewInt(10))
+	assert.Error(err, "should have error because of the wrong nonce")
+	//wrong epoch
+	header.Nonce = block.IHeader().GetNonce()
+	err = q.VerifySeal(nil, header, big.NewInt(1000))
```

Comment:
why change diff?

Reply:
test VerifySeal  to show  err        maybe add instroduce about it

Reply:
it's about  epoch, so change height as python does

Reply:
impossible and meaningless to change height without change header. remove the epoch case.